Gruntfile.js

```diff
@@ -447,7 +448,7 @@ module.exports = (grunt) => {
     const fork2 = function () {
       grunt.log.writeln('backend: forking...')
       child = fork(backendIndex, process.argv, {
-        env: { NODE_ENV, ...process.env },
+        env: { CHEL_AVOID_DECRYPTION_ATTEMPTS: "'true'", NODE_ENV, ...process.env },
```

Comment:
Would it be possible to somehow set this inside of the server code itself instead of here?

It's really the server that's saying, "I don't want Chelonia to try to decrypt messages", not Grunt.

Reply:
I looked into this, using `config`. But since encryptedData is outside of Chelonia, doing it this way (with an env variable) seemed reasonable, which is probably why it was done like so previously.

> It's really the server that's saying, "I don't want Chelonia to try to decrypt messages", not Grunt.

In a way, but then this is Grunt that's executing the dev server, and in this case `CHEL_AVOID_DECRYPTION_ATTEMPTS` should be set to true. With PR #2759 there could be more flexibility to set or not this.

Reply:
Thinking about this, I think that using `config` is superior, but we'd need to make more changes than just renaming an env variable. The changes would need to be done _in Chelonia_ every time `unwrapMaybeEncryptedData`. For example, `const wv = this.config.someNameToBeDefined ? unwrapMaybeEncryptedData(v) : undefined`.

Reply:
> I mean it's a utility library that's not got (easy) access to the same this that Chelonia does. Even though, yes, technically, it shares a directory.

I know this is an ugly hack, but could you in that case set a global variable? It would avoid the need for uglifying the code with this outer-check. The check would be able to be left in-place.

Reply:
I think that what we're doing currently is an ugly hack that happens to be convenient, but using a global variable for this just seems wrong. I think that this mostly could be handled by Chelonia, even without such a check.